nipype/interfaces/mrtrix3/preprocess.py

```diff
@@ -241,6 +242,151 @@ def _format_arg(self, name, trait_spec, value):
                 return f"-{trait_spec.argstr}"
         return super()._format_arg(name, trait_spec, value)
 
+
+class DWIPreprocInputSpec(MRTrix3BaseInputSpec):
+    in_file = File(
+        exists=True, argstr="%s", position=-10, mandatory=True, desc="input DWI image"
+    )
+    out_file = File(
+        "preproc.mif",
+        argstr="%s",
+        mandatory=True,
+        position=-9,
+        usedefault=True,
+        desc="output file after preprocessing",
+    )
+    rpe_options = traits.Enum(
+        "-rpe_none",
+        "-rpe_pair",
+        "-rpe_all",
+        "-rpe_header",
+        argstr="%s",
+        position=-8,
+        mandatory=True,
+        desc="Specify acquisition phase-encoding design, one of the -rpe_* options must be provided",
```

Comment:
I would need to go read some external documentation to use this. Any chance of a more helpful description?

Reply:
What about:
```suggestion
        desc="Specify acquisition phase-encoding design. "none" for no reversed phase-encoding image, "all" for all DWIs have opposing phase-encoding acquisition, "pair" for using a pair of b0 volumes for inhomogeneity field estimation only, and "header" for phase-encoding information can be found in the image header(s)",
```